pages/projects/precycle.md

```diff
@@ -0,0 +1,34 @@
+---
+  layout: code_for_new_orleans_project
+  title: Precycle
+  category: project
+  project:
+    announce: Want to contribute to this project? <a href='mailto:samantha@hackshackers.com'>Contact the Project Champion!</a>
+    champion:
+      name: Samantha Sunne
+      email: samantha@hackshackers.com
+    people:
+      -
+        name: Samantha Sunne
+        title: Data Lead
+      -
+        name: Jonathon Hooper
+        title: Code Lead
+    goals:
+      - Create a site that shows New Orleanians where and how to recycle
+      - Create a subscribe-able calendar that will remind residents to recycle
+      - Fix the error that is returning [object Object] on the [live site](http://precycle.jonathanhooper.net/)
+      
+    skills_needed:
+      - jQuery
+      - JavaScript
+    why: |
+      In New Orleans, it can be hard to tell what you are allowed to recycle - glass? cardboard? Mardi Gras beads?
+      
+      If New Orleanians successfully recycled their materials, it would be better for the environment. This site returns the date of recycling pickup and the materials that can be turned in. It also creates a Google Calendar that reminds the user when to recycle and where.
+    where: |
+      The prototype is on Jonathan's [website](http://precycle.jonathanhooper.net/).
+
+     The code is [on Github](https://github.com/jmhooper/precycle).
```

Comment:
Hey @samanthasunne, I tried to get this page to compile but it needs one more space at the front of this line to format correctly.  Could you make this change please?